Helper/Marketplace/WebkulHelper.php

```diff
@@ -118,19 +146,97 @@ private function getSellerData($itemPrice, $sellerDetail)
         }
 
         return [
-            "sellerCommission" => $sellerCommission,
+            "marketplaceCommission" => $marketplaceCommission,
+            "commission" => $sellerCommission,
             "pagarmeId" => $recipient['pagarme_id']
         ];
     }
 
-    private function getTotalSellerCommission(array $splitData)
+    private function getTotalSellerCommission(array $sellersData)
     {
         $totalCommission = 0;
 
-        foreach ($splitData as $commission) {
-            $totalCommission += $commission['sellerCommission'];
+        foreach ($sellersData as $commission) {
+            $totalCommission += $commission['commission'];
         }
 
         return $totalCommission;
     }
+
+    private function setRemainderToResponsible($remainder, $splitData)
+    {
+        $responsible = $this->moduleConfig
+            ->getMarketplaceConfig()
+            ->getResponsibilityForReceivingSplitRemainder();
+
+        $splitData = $this->floatToCentsSplitData($splitData);
+
+        if ($responsible == 'marketplace') {
```

Comment:
Aqui a gente poderia criar constantes pra os valores `'marketplace'`,` 'marketplace_sellers'` e `'sellers'` e fazer um switch, tipo isso aqui:

```suggestion
        const ONLY_MARKETPLACE = 'marketplace';
        const MARKETPLACE_SELLERS = 'marketplace_sellers;
        const ONLY_SELLERS = 'sellers'
        switch ($responsible) {
          case self::ONLY_MARKETPLACE:
           $splitData['marketplace']['totalCommission'] += $remainder;
             return $splitData;
           case self::ONLY_SELLERS:
             return $this->divideRemainderBetweenSellers(
                $remainder,
                $splitData
            );
          case self::MARKETPLACE_SELLERS:
             return $this->divideRemainderBetweenMarkeplaceAndSellers(
                $remainder,
                $splitData
            );
}
```

Reply:
Adicionado!